learning/tour-of-beam/frontend/lib/models/event_context.dart

```diff
@@ -18,27 +18,31 @@
 
 import 'package:equatable/equatable.dart';
 
-class AnalyticsEvent with EquatableMixin {
-  final String action;
-  final String category;
-  final String? label;
-  final Map<String, String>? parameters;
-  final int? value;
+const _none = 'none';
 
-  AnalyticsEvent({
-    required this.action,
-    required this.category,
-    this.label,
-    this.parameters,
-    this.value,
+/// Basic information of the Tour of Beam state to augment analytics events.
```

Comment:
What Tour of Beam state?  The SdkId and the UnitId?

Reply:
Currently yes. I do not want to spell this out in the comment because if we add more parameters in the future (like UI locale) we will forget to update this comment and it will contain an incomplete list.